SpatialGDK/Source/SpatialGDK/Public/Utils/SchemaDatabase.h

```diff
@@ -12,13 +12,13 @@ struct FSubobjectSchemaData
 {
 	GENERATED_USTRUCT_BODY()
 
-	UPROPERTY(VisibleAnywhere)
+	UPROPERTY(Category = "SpatialGDK", VisibleAnywhere)
```

Comment:
Category's are required for when the UnrealGDK is installed as an Engine Plugin (which is now an option and is default with the new InstallGDK.bat script in the Engine.